builtin/providers/google/resource_container_cluster.go

```diff
@@ -142,7 +142,56 @@ func resourceContainerCluster() *schema.Resource {
 				Default:  "default",
 				ForceNew: true,
 			},
-
+			"subnetwork": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"addons_config": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				MaxItems: 1,
```

Comment:
I'm glad `MaxItems` is finally a field! This should definitely make its way into the rest of the Google provider at some point...